datafusion/core/src/datasource/file_format/mod.rs

```diff
@@ -84,7 +84,7 @@ pub trait FileFormat: Send + Sync + fmt::Debug {
         &self,
         state: &SessionState,
         conf: FileScanConfig,
-        filters: &[Expr],
+        filters: Option<&Arc<dyn PhysicalExpr>>,
```

Comment:
I think it is a good change to have this take a single `PhysicalExpr` rather than a slice as it is more consistent with `TableProvider::scan` 👍